### PR TITLE
nbeat multivariate change and unittest for multivariate (succinct one)

### PR DESCRIPTION
**Pull request by dumjax**
Fixes #.

### CODE REVIEW

File: darts/tests/test_NBEATS.py [hunk 1]
```diff
@@ -45,8 +46,16 @@ def test_fit(self):
...
+            series_multivariate = tg.linear_timeseries(length=100).stack(tg.linear_timeseries(length=100, start_value = 0, end_value=0.5))
+            model = NBEATSModel(input_chunk_length=1, output_chunk_length=1, n_epochs=20)
+
+            model.fit(series_multivariate)
+            res = model.predict(n=2).values()
+            # the theoretical result should be [[1.01, 1.02], [0.505, 0.51]].
+            # We just test if the given result is not too far in average.
+            self.assertTrue(abs(np.average(res-np.array([[1.01, 1.02], [0.505, 0.51]])) < 0.02))
+
+            # with self.assertRaises(ValueError):
```
Comment (hrzn) [inline darts/tests/test_NBEATS.py:60]: You can remove these lines.

File: darts/tests/test_NBEATS.py [hunk 2]
```diff
@@ -45,8 +46,16 @@ def test_fit(self):
...
         def test_multivariate(self):
-            series_multivariate = tg.linear_timeseries(length=100).stack(tg.linear_timeseries(length=100))
-            model = NBEATSModel(input_chunk_length=1, output_chunk_length=1,
-                                n_epochs=10, num_stacks=1, num_blocks=1, layer_widths=20)
-            with self.assertRaises(ValueError):
-                model.fit(series_multivariate)
+
+            # testing a 2-variate linear ts, first one from 0 to 1, second one from 0 to 0.5, length 100
+            series_multivariate = tg.linear_timeseries(length=100).stack(tg.linear_timeseries(length=100, start_value = 0, end_value=0.5))
+            model = NBEATSModel(input_chunk_length=1, output_chunk_length=1, n_epochs=20)
```
Comment (hrzn) [inline darts/tests/test_NBEATS.py:52]: how about maybe test a slightly less trivial version of the model here? For instance with `input_chunk_length=3` or something like that? I fear that values of 1 may lead to corner cases.

File: darts/models/nbeats.py [hunk 3]
```diff
@@ -279,26 +281,48 @@ def __init__(self,
...
+                                       self.input_chunk_length_multi,
+                                       self.target_length,
+                                       _GType.SEASONALITY)
             self.stacks_list = [trend_stack, seasonality_stack]
 
         self.stacks = nn.ModuleList(self.stacks_list)
 
     def forward(self, x):
 
+        # if x1, x2,... y1, y2... two multivariate ts, and a1, a2... one covariate ts
```
Comment (hrzn) [inline darts/models/nbeats.py:323]:
```suggestion
        # if x1, x2,... y1, y2... is one multivariate ts containing x and y, and a1, a2... one covariate ts
```

**Review by hrzn — APPROVED** [review]
LGTM
